### ISSUE TITLE
[Idea] nerf default weapons

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/gameplay#53]
I think default weapons are too strong. It's not rare to see people prefer to stay dretch or naked+smg instead of using, say, mantis or lasgun (shotgun does not count, as this weapon is very powerful... maybe it should also be nerfed, but I have not thought about it yet).

To give some numbers:

* smg kills a dretch with 6 shots, 80ms per shot, 0.5s to kill
* dretch kill naked human in 2 head shots, 1s to kill
* 3 smg kill a tyrant in 1.9s (a better number would be for 2 smg, but this implies a reload and I'm lazy. Consider the fact 2 smg vs 1 rant is basically when tyrant attacks a base, which usually have turrets, so it's even faster than what I describe)
* 3 dretches kill a BS in 2.3s (here, the real situations tend to be more about a longer time, because it's much harder to do head shots on a BS and alien defenses are mostly useless)

**Disclaimer**
>Ideas without a related problem issue and lacking explanation or depth are closed after a certain time.

**Comment by bmorel** [comment]
My proposal would be to reduce dretch damages from 36 to 32, and SMG damages from 5 to 4, remove 5 ammo from clips, and add it 1 clip.
It would result in the following numbers:

* smg kills a dretch with 8 shots, 80ms per shot, 0.6s to kill
* dretch kill naked human in 3 head shots, 1.5s to kill
* 3 smg kill a tyrant in 2.4s... if they still had 30 ammo per clip. In practice, it would be longer thanks to the 5 ammo/clip reduction (implies a reload).
* 3 dretches kill a BS in 2.6s

I am  not entirely satisfied by the resulting situation, but I think it would lead to an improvement, and would reduce the need to make end-classes stronger.

**Comment by DolceTriade** [comment]
I think this makes sense and we should experiment with this. But I don't think these changes can work alone, they need to be implemented with an alternative means of earning evos/creds.

**Comment by bmorel** [comment]
There's already the reward for destroying that made it into master (disabled by default)

**Comment by Gireen** [comment]
I don't think one can compare dretch and smg just by dps
They are close and long range weapons.
Aliens have to follow the target to deal damage and there are also different damage zones. 
I don't think constant headshots or bites are a given.

Besides that does this idea also break the balance between dretch and smg in its own logic.
It is a much bigger nerf for the dretch.

**Comment by DolceTriade** [comment]
tbh, any damage nerf for the dretch doesn't really matter as long as naked humans are killed in 2 hits right? The idea is that dretch swarms are too strong against more expensive humans and they really shouldn't be.

**Comment by necessarily-equal** [comment]
For the record, @Gireen's mod had naked humans SMG bullets a bit more harmfull with 6 dmg instead of 5. I found it made some sense while playtesting because aliens can pretty strongly starve humans of their credits. Not sure about how it would play out, but I actually wanted to buff SMG to nerf it a bit against adv goon and tyrant (and tyrant would be doubly resistant to it with the ["bulletproof tyrant" idea](https://github.com/Unvanquished/gameplay/issues/45)).

(for reference, my todo-list for my balancing ideas https://paste.gnugen.ch/paste/9gbQ)

**Comment by Viech** [comment]
This looks like the kind of issue we wanted to prevent by having categories for discussion. It states a perceived problem, which is pretty complex in nature and needs focused discussion to determine whether the impression stated (players temporarily use weaker equipment to save credits) even is bad in any way, and the issue _also_ gives a very specific solution idea that should be discussed separately (when it turns out that a change is desired). Imo this issue should be turned into a problem to discuss that part first.

**Comment by sweet235** [comment]
There are multiple reasons a player can have for using a dretch while having enough morph points for higher classes. As has been said, one reason is the plan to save up some more morph points. I don't think this is bad in any way.

If you see me using a dretch in such cases, other reasons might be:

- I want to attack humans without drawing too much attention, maybe while making them think I am a poor bot
- I try to sneak past some human defence line, and evolve there
- I am a complete noob with mantis, and not very good with marauder

**Comment by Viech** [comment]
> I am a complete noob with mantis, and not very good with marauder

I'm probably one of the few players to use the Mantis since I've spent countless hours fine-tuning its pounce-ability and feel pretty comfortable with it; so I can see that maybe it isn't appealing enough to spend morphs on for most.

I also don't think that it's bad to be able to save evos. In particular, the flip side of the coin would be to feel useless and that you cannot progress when you do not _have_ morphs/credits to spend.

**Comment by sweet235** [comment]
> The idea is that dretch swarms are too strong against more expensive humans and they really shouldn't be.

Maybe this is a problem related to bots. I agree that an army of 3 dretch bots seems more dangerous than it should be. But I try to keep in mind that these 3 dretches are locked in attacking a single human, so human forces are free to do other things. If we consider a game without any bots, having 3 alien players locked in an attack on a single human should give humans a big tactical advantage.

> In particular, the flip side of the coin would be to feel useless and that you cannot progress when you do not have morphs/credits to spend.

In fact I heard complaints about that more than once: players already feel helpless when fighting battlesuits as dretches or tyrants as naked human. I don't think the current balance is too bad regarding this.

> I can see that maybe it isn't appealing enough to spend morphs on for most.

I think manits is quite strong compared to its cost. It is just that I lack practice using it.

**Comment by VReaperV** [comment]
Rifle is better in DPS than shotgun/lasgun/mass driver, but only if you don't include reload. Once you include even one reload, only MD has lower DPS than the rifle. You also need to account for spread. In regards to time to kill for each alien class, technically for Adv Mara, Goon, Adv Goon and Tyrant, every other weapon outclasses rifle with one exception (MD vs Adv Goon). But considering spread, you'd need 100% accuracy at close range for it to be better than lasgun and MD vs Mara, once you add a reload it becomes significantly worse than other weapons. (I'm not including blaster, grenade and firebomb for obvious reasons, and the flamthrower because I didn't look at its numbers after the nerf)

Also, rifle only outclasses lasgun vs Mantis, very slightly outclasses lasgun and MD vs granger/adv granger (and technically chaingun vs granger), and only outclasses lasgun vs dretch.

**Comment by bmorel** [comment]
I think I should have indeed made 2 tickets for this: one for rifle, and one for dretch.

Anyway, about SMG/rifle: I've seen them be very deadly in defense, notably against tyrants, and I do lot of money using that to kill a fleeing goon. All it requires is starting firing when enemy runs away.

**Comment by VReaperV** [comment]
Yeah, but you need to consider the other effects of weapon/class stat changes. E. g. a rifle with 25x4 dmg per magazine will not be able to detroy an acid tube without reloading. It also means that you'd most likely need to reload twice to kill a goon, and definitely at least twice vs adv goon, and 3 times vs a tyrant, making it rather useless.

I think advanced classes and armor dying fast in an enemy base is more the result of there being more defenses (both players/bots and structures) focusing on whoever is attacking. So one attacker would usually get melted quickly no matter the class/armor. Changing that would mean changing the encounters outside bases, unless the change only affects something within a base.

**Comment by bmorel** [comment]
> and definitely at least twice vs adv goon, and 3 times vs a tyrant, making it rather useless.

But it should be useless against such strong classes, imo (especially for rants). Or if not, then dretch should be buffed, to be useful against BS (or BS nerfed?). Which goes against my 1st proposal.

> So one attacker would usually get melted quickly no matter the class/armor. 

This is sadly not true when it comes to attacking alien bases. And on human bases, stage 0 and stage 1 defenses can destroy a stage 4 class before it dealt any damage.

I know it's not an easy topic, though, and it's also why I initially proposed a nerf for both starting classes, because I don't believe changing only a single weapon can lead to good results.

**Comment by VReaperV** [comment]
> But it should be useless against such strong classes, imo (especially for rants).

Then you will have no way of defending or attacking if your team is out of credits/evos. If a team is gettinng rushed with rants/suits while they only have rifles/dretches, that usually means that the defending team is close to losing. Being able to do enough damage gives them a chance to fight back.

> This is sadly not true when it comes to attacking alien bases. And on human bases, stage 0 and stage 1 defenses can destroy a stage 4 class before it dealt any damage.

I agree that alien defenses are worse if there's no players/bots defending. But aliens also have more mobility and high-range aliensense to come back to the base to defend it.

**Comment by bmorel** [comment]
> I agree that alien defenses are worse if there's no [...] bots defending. But aliens also have more mobility and high-range aliensense to come back to the base to defend it.

I removed the "player" word here on purpose: I consider bots important, and I wanted to bring on the fact that the aliensense high range is actually why currently alien bots are less  good than human ones.
